### PR TITLE
Refactor squeeze prim to accept a sequence of dimensions.

### LINKED ISSUE #2421: squeeze prim should accept a sequence of dimensions to squeeze, not just one
Ex. `prims.squeeze(a, a.shape, (0, 1))`

### PR DESCRIPTION
**Pull request by rdspring1**
Fixes #2421

### CODE REVIEW

**Review by kevinstephano — CHANGES_REQUESTED** [review]
Looks good.  Could you also increment the `version.txt` file contents from `0.0.1` to `0.0.2`.

**Comment by kevinstephano** [comment]
@jjsjann123 ^^^^ is that the proper way to change the version?

File: third_party/nvfuser/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-0.0.1
+0.0.2
```
Comment (jjsjann123) [inline third_party/nvfuser/version.txt:1]: Muhahahahaha, now we have to make sure that no one else is bumping the version on other inflight PRs.

Human mutex is on! @kevinstephano

File: third_party/nvfuser/csrc/python_frontend/fusion_record.h [hunk 2]
```diff
@@ -443,14 +443,14 @@ struct SqueezeOpRecord : RecordFunctor {
       std::vector<State> _args,
       std::vector<State> _outputs,
       std::vector<int64_t>& original_shape,
-      int64_t dim)
+      std::vector<int64_t>& dims)
```
Comment (jjsjann123) [inline third_party/nvfuser/csrc/python_frontend/fusion_record.h:446]: So with this change, we can no longer accept a single number for `dim` any more? This is getting ugly....
I'm advocating to push our user facing API from pybind to real python scripts, so these can be easily handled in python, other than the cumbersome pybind overload.
https://github.com/csarofeen/pytorch/pull/2430#issuecomment-1423037712

Comment (jjsjann123) [reply third_party/nvfuser/csrc/python_frontend/fusion_record.h:446]: Just to make it clear, not suggesting that we should do it in this PR. We can merge it as-is

**Review by jjsjann123 — APPROVED** [review]
LGTM

**Review by kevinstephano — APPROVED** [review]
LGTM!

**Pull request merged**